_includes/pbc/all/install-features/202311.0/install-the-shipment-product-offer-service-points-availability-feature.md

```diff
@@ -0,0 +1,81 @@
+
+| NAME                                      | VERSION          | INTEGRATION GUIDE                                                                                                                                                                                                      |
+|-------------------------------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Product Offer Service Points Availability | {{page.version}} | [Install the Product Offer Service Points Availability feature](/docs/pbc/all/service-points/{{page.version}}/unified-commerce/install-and-upgrade/install-the-product-offer-service-points-availability-feature.html) |
+| Shipment                                  | {{page.version}} | [Shipment feature integration](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/install-and-upgrade/install-features/install-the-shipment-feature.html)                                                     |
+
+## 1) Install the required modules
+
+Install the required modules using Composer:
+
+```bash
+composer require spryker-feature/shipment-product-offer-service-points-availability: "{{page.version}}" --update-with-dependencies
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following modules have been installed:
+
+| MODULE                                                    | EXPECTED DIRECTORY                                                               |
+|-----------------------------------------------------------|----------------------------------------------------------------------------------|
+| ShipmentTypeProductOfferServicePointAvailabilitiesRestApi | vendor/spryker/shipment-type-product-offer-service-point-availabilities-rest-api |
+
+{% endinfo_block %}
+
+
+## 2) Set up transfer objects
+
+Generate transfer changes:
+
+```bash
+console transfer:generate
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure the following changes have been applied in transfer objects:
+
+| TRANSFER                                                    | TYPE  | EVENT   | PATH                                                                                              |
+|-------------------------------------------------------------|-------|---------|---------------------------------------------------------------------------------------------------|
+| RestProductOfferServicePointAvailabilitiesRequestAttributes | class | created | src/Generated/Shared/Transfer/RestProductOfferServicePointAvailabilitiesRequestAttributesTransfer |
+| ProductOfferServicePointAvailabilityConditions              | class | created | src/Generated/Shared/Transfer/ProductOfferServicePointAvailabilityConditionsTransfer              |
+
+{% endinfo_block %}
+
+## 3) Set up behavior
+
+{% info_block warningBox "Verification" %}
```

Comment:
There are no steps in this section. What are we verifying?

Reply:
@andriitserkovnyi, we are verifying that the filter can be used with the resource in Storefront API and there is only one step - `POST https://glue.mysprykershop.com/product-offer-service-point-availabilities`.

Reply:
@romanhavrylko, I mean that there are no steps in this section besides verification.

Reply:
@andriitserkovnyi, updated.